### DISCUSSION TITLE
【0x10】Deploy your Voxel dApp on Github pages based on Polygon Test Network!

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#86] [Projects]
For Example:

> https://leeduckgo.github.io/Tai-Shang-Voxel-Handler-V2/

<img width="1439" alt="image" src="https://user-images.githubusercontent.com/12784118/168469294-20c43883-393c-422a-8b7b-4615682d5cb0.png">

**Comment by alertcat** [comment]
![image](https://user-images.githubusercontent.com/73512451/168471086-14ce80aa-ceac-4ec6-b1f0-34f777ed368e.png)
